Cargo.toml

```
@@ -21,12 +21,12 @@ include = [
 edition = "2021"
 
 [dependencies]
-embedded-hal = "0.2"
+embedded-hal = "1"
 rtcc = "0.3"
 
 [dev-dependencies]
-linux-embedded-hal = "0.3"
-embedded-hal-mock = "0.8"
+linux-embedded-hal = "0.4"
+embedded-hal-mock = "0.11.1"
```

```suggestion
embedded-hal-mock = { version = "0.11.1", default-features = false, features=["eh1"] }
```
So that we avoid depending on embedded-hal 0.2

---

This has caused a weird issue with CI, but I ran this command locally and it was fine:
```
cargo build --target=arm-unknown-linux-gnueabi
```

---

Ok the problem is that this is split over several lines in your version. In CI the line is commented out depending on the target, resulting in a broken toml file. If you put it into one line it should work.